### PR TITLE
Pull request related to repo topic tab 

### PR DESCRIPTION
**Pull request by uidev547**

### CODE REVIEW

**Comment by uidev547** [comment]
demo page url:
http://uidev547.github.io/KodeBeagle/search
and 
settings url : http://192.168.2.140:9201

**Comment by hashd** [comment]
Too many commits for a single pull request, squash it, perhaps?

File: _includes/search-result/files-content.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+    
+    <div class="row">
+        <div class="col-sm-9 editor-section-wrapper">
+            <div ng-if="!model.filterNotFound">
+              <div ng-if="model.editors && model.editors.length>0" data-step="3" data-intro="Results">
+                  <div ng-class="{'file-list-view': !model.toggelSnippet }">
```
Comment (hashd) [inline _includes/search-result/files-content.html]: toggleSnippets is misspelled here

File: _includes/search-result/repotopic.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+        <div ng-if="model.repoRequest=='responded' && !model.repos.length">
+            <div class="text-center jumbotron">
+                no matches found
+            </div>
+        </div>
+    </div>
+    {% endraw %}
+</tab>
+<style type="text/css">
```
Comment (hashd) [inline _includes/search-result/repotopic.html]: Why is styling included this way here?

File: library/dev/js/controller/editors-controller.js [hunk 3]
```diff
@@ -117,9 +117,95 @@
...
+                  obj.reponame = ( function ( url ) {
+                    url = url.split('/');
+                    return url[0] + '/' + url[1];
+                  } )( obj.files[0].fileName );
+                  obj.terms = res[i]._source.terms.slice( 0, 10 );
+                  obj.repoStars = +res[i]._source.repository.repoStars;
+                  obj.repoId = res[i]._source.repository.repoId;
+                  repos.push( obj );  
+                }
+
```
Comment (hashd) [inline library/dev/js/controller/editors-controller.js]: Do not use 2 consecutive blank lines

File: library/dev/js/factories/docs-service-factory.js [hunk 4]
```diff
@@ -453,6 +473,33 @@
           linesData: getlinesObj( fileContent, fileInfo.lines, offset )
         }
       }
+
+      var searchRepotopic = function  ( obj ) {
+        
+        var correctedQuery
```
Comment (hashd) [inline library/dev/js/factories/docs-service-factory.js]: Use trailing comma here

File: library/dev/js/vendors/angular-code-mirror.js [hunk 5]
```diff
@@ -1123,7 +1228,7 @@ var prettyPrint;
    * @param {boolean} isPreformatted true iff white-space in text nodes should
    *     be treated as significant.
    */
-  function numberLines(node, opt_startLineNum, isPreformatted, opt_heilightLines) {
+  function numberLines(node, opt_startLineNum, isPreformatted, opt_heilightLines, opt_heilightLines_obj) {
```
Comment (hashd) [inline library/dev/js/vendors/angular-code-mirror.js]: Avoid mix mash of snake and camel casing.
Highlight is spelled incorrectly here too.

**Comment by hashd** [comment]
FYI, the changes are so huge, I see a usability issue on Github which can probably be reported

**Comment by sachintyagi22** [comment]
Closing this as it is replaced by other PR #472

**Pull request closed without merging**